src/test/java/org/openelisglobal/testanalyte/TestAnalyteServiceTest.java

```diff
@@ -0,0 +1,141 @@
+package org.openelisglobal.testanalyte;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+import org.openelisglobal.BaseWebContextSensitiveTest;
+import org.openelisglobal.testanalyte.service.TestAnalyteService;
+import org.openelisglobal.testanalyte.valueholder.TestAnalyte;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class TestAnalyteServiceTest extends BaseWebContextSensitiveTest {
+
+    @Autowired
+    private TestAnalyteService testAnalyteService;
+
+    @Before
+    public void setUp() throws Exception {
+        executeDataSetWithStateManagement("testdata/test-analyte.xml");
+    }
+
+    @Test
+    public void testDataInDataBase() {
+        List<TestAnalyte> testAnalytes = testAnalyteService.getAll();
+        testAnalytes.forEach(testAnalyte -> {
+            System.out.print(testAnalyte.getId() + " - " + testAnalyte.getTest().getId() + " - "
```

Comment:
am merging this but create a followup commit to remove this line printing on the console